### ISSUE TITLE
Training Loss Clarify

### ISSUE THREAD

**Issue by chuong98** [xingyizhou/CenterNet2#33]
## 📚 Documentation

I check over, and have a bit confused about the loss implementation. Take the ATSS as RPN as an example:
1. In the first stage, training the RPN with Focal Loss and IoU loss.
2. In the second stage, training with the Cross Entropy with Softmax output. 

My question is about the Second Stage:
1. Do you still need to predict the Box Regression, since the First Stage already does a very good job?
2. Do you train on the product `(P(Obj)*P(Cls)) `in the second stage ? Or Only on` P(Cls)` as in conventional Faster-RCNN? 

In other word, the training losses are identical to Faster-RCNN. But during inference, you use P(Obj)*P(Cls) as final output ?

**Comment by nilskk** [comment]
I would also like to know the answer :)

**Comment by jshilong** [comment]
> ##  Documentation
> Hi, thanks for the paper:
> I am a bit confused about the loss implementation. Take the ATSS as RPN as an example:
> 
> 1. In the first stage, training the RPN with Focal Loss and IoU loss, identical to ATSS.
> 2. In the second stage, training with the Cross Entropy with Softmax output.
> 
> My question is about the Second Stage:
> 
> 1. Do you still need to predict the Box Regression, since the First Stage already does a very good job?
> 2. For the classification, do you train on the product `(P(Obj)*P(Cls)) `in the second stage ? Or Only on` P(Cls)` as in conventional Faster-RCNN?
> 
> In other word, the training losses are identical to Faster-RCNN. But during inference, you use P(Obj)*P(Cls) as final output ?

You may find the answer in #15
